Changes

```diff
@@ -2314,6 +2314,9 @@ OCaml 4.14 maintenance version
   (Xavier Leroy and Anil Madhavapeddy, review by Anil Madhavapeddy and
   Sébastien Hinderer)
 
+- #12930: Disable control flow integrity on OpenBSD >= 7.4.
+  (Michael Hendricks, Miod Vallat, review by Gabriel Scherer)
```

Comment:
Actually (sorry to nitpick) I didn't review this work myself (I am not qualified to vouch for it), I just turned the "gray-approval" stamp of Miod into a "green-approval" stamp.

Reply:
Thank you. I appreciate the nitpick. I see no reason to allow sloppy work :-) I've changed it to show Miod as the reviewer.